### PR TITLE
feat: NOD-703 - BlockfabricBridge: Key validation

### PR DESCRIPTION
**Pull request by jsanders5**
This pull requests implements a summary and validation package to display all attainable smart contract information before and after a contract is deployed. The validation package will decypt all keystore files generated for the bridge deployment, and ensure the address matches what is stored on-chain. 
- Updated the organization for `neptune` from `elevateunited' to `NerdCoreSdk`

### CODE REVIEW

File: bridge_deployer/common/common.go [hunk 1]
```diff
@@ -78,3 +90,184 @@ func GetAddressFromPrivateKey(addressPrivateKey string) (common.Address, error)
...
+}
+
+//func scGetter(instance *lockupV1.Lockup) (*big.Int, error) {
+//	ret, err := instance.GetterFunc(nil)
+//	if err != nil {
+//		log.Fatal(err)
+//	}
+//
+//	return ret, err
+//}
```
Comment (adamtew) [inline bridge_deployer/common/common.go:235]: Do we want to keep this commented code?

File: ceremony-scripts/deploy_bridge_contracts.sh [hunk 2]
```diff
@@ -89,65 +90,77 @@ get_deployer_a_private_key() {
...
+
+    mv bridge_address ${VOLUMES_DIR}/volume5/bridge_address
 
-    echo "bridge output=" $bridge_output &>> ${LOG_FILE}
-    bridge_address="$(echo $bridge_output | tail -n1)"
-    echo $bridge_address > ${VOLUMES_DIR}/volume5/bridge_address
+    printer -n "L2 Bridge Deployed."
+ }
 
+ deploy_token() {
```
Comment (adamtew) [inline ceremony-scripts/deploy_bridge_contracts.sh:111]: Nice, I like the modularity of moving these into separate functions.

File: ceremony-scripts/deploy_bridge_contracts.sh [hunk 3]
```diff
@@ -89,65 +90,77 @@ get_deployer_a_private_key() {
...
-        ${notary_address} \
-        ${token_contract_address} \
-        ${CHAIN_ID})"
-
-    bridge_minter_address="$(echo $bridge_minter_output | tail -n1)"
-    echo $bridge_minter_address > ${VOLUMES_DIR}/volume5/bridge_minter_address
+        $1\
+        $2 \
+        $3 \
+        $4 \
```
Comment (adamtew) [inline ceremony-scripts/deploy_bridge_contracts.sh:135]: Same thing here with named vs positional params

File: bridge_deployer/common/common.go [hunk 4]
```diff
@@ -78,3 +90,184 @@ func GetAddressFromPrivateKey(addressPrivateKey string) (common.Address, error)
...
+		nil,
+		nil,
+		nil,
+		nil,
+	)
+
+	return cfg, err
+}
+
+func GetAdminKey(path string) (string, error) {
```
Comment (adamtew) [inline bridge_deployer/common/common.go:119]: Is this used?

**Review by jmbrown412 — APPROVED** [review]
lgtm

**Pull request merged**